<!--xml version = "1.0"-->
<launch>
    <node name="bag2tum" pkg="bag2tum" type="bag2tum_node" output="screen">
        <param name="save_folder" value="/home/da/data/hlaf" />
        <param name="rgb_topic" value="/kinect2/qhd/image_color_rect" />
        <param name="depth_topic" value="/kinect2/qhd/image_depth_rect" />
        <param name="thermal_topic" value="/optris/thermal_image" />
<!--         <param name="gt_topic" value="/vrpn_client_node/RigidBody/pose" /> -->
    </node>

</launch>
